_posts/2020-03-16-sr2020-competition-postponed.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: news
+title: Student Robotics 2020 Postponed
+---
+
+Following guidance from the UK government and the University of Reading, **we will need to postpone Student Robotics 2020** from the original date in April, until some time late June / early July.
+
+We are currently investigating the following dates for the competition:
+
+- 27th - 28th June
+- 4th - 5th July
+- 11th - 12th July
+
+The event will still take place over a weekend, at the University of Reading.
+
+We have requested feedback from team leaders and volunteers to help select the most appropriate date.
+
+We hope to confirm the updated date in the next week, so please let us know as soon as you can.
```

Comment:
On second thoughts - is this better to be kept in just the email?

Reply:
Ooh, good catch!

```suggestion
We hope to confirm the updated date in the next week, so watch this space!
```